jekyll/_cci2/testing-ios.md

```diff
@@ -297,79 +186,94 @@ workflows:
             - build-and-test
 ```
 
-The environment variable `FL_OUTPUT_DIR` is the artifact directory where FastLane logs should be stored. Use this to set the path in the `store_artifacts` step to automatically save logs such as Gym and Scan.
+The environment variable `FL_OUTPUT_DIR` is the artifact directory where FastLane logs, and signed `.ipa` file should be stored. Use this to set the path in the `store_artifacts` step to automatically save logs and build artifacts from Fastlane.
```

Comment:
```suggestion
The environment variable `FL_OUTPUT_DIR` is the artifact directory where FastLane logs and signed `.ipa` files should be stored. Use this to set the path in the `store_artifacts` step to automatically save logs and build artifacts from Fastlane.
```

Reply:
@rosieyohannan This should be singular as the output will just be one `.ipa` file. I'll remove the comma in another commit